examples/core/core_2d_camera.ml

```diff
@@ -42,19 +42,20 @@ let rec loop (player, buildings, colors, camera) =
   | true -> Raylib.close_window ()
   | false ->
       let open Raylib in
-      ( if is_key_down Key.Right then Rectangle.(set_x player (x player +. 2.0))
-      else if is_key_down Key.Left then
+      ( if is_key_down KeyboardKey.Right then
```

Comment:
I'm a bit torn on this. It is certainly closer to the C version, but it is also something that will be typed a lot, especially starting out. Since there are not other `Key`s in raylib, I suggest we keep the short version.

Reply:
Fixed.